backend/src/Edna/Config/Definition.hs

```diff
@@ -69,6 +72,8 @@ data LoggingConfig =
   -- ^ No logging.
   deriving stock (Generic, Show, Eq)
 
+type MdeHost = String
```

Comment:
By default we prefer `Text` to `String` and use `String` only where it's required by external API or there is a solid reason to use it.

Reply:
I thought about it, but in task there:
> Add optional configuration parameter mde-link, its type is string
I'll change it to Text)

Reply:
Oh, I meant abstract string type, not specifically Haskell's `String` :slightly_smiling_face: So it's string from the user's standpoint as the user doesn't care what actual type used in Haskell, users just pass it as command line option/env var/config field.